pkgs/by-name/ri/ride/package.nix

```diff
@@ -0,0 +1,119 @@
+{ lib
+, stdenv
+, buildNpmPackage
+, fetchFromGitHub
+, substituteAll
+
+, jq
+, moreutils
+, zip
+, makeWrapper
```

Comment:
For a better compatibility with Darwin arches, it is preferred to use `makeBinaryWrapper` instead.

Reply:
Does the normal shell-type `makeWrapper` just not work on Darwin? Or is there some more complicated reason behind it?

Reply:
It works, but on some particular case, it creates issues. I've already fixed some pull requests thanks to that, and now by default I use it. It's also faster since it's creating a C binary.

Reply:
I guess it can be a good default. My only issue with the binary wrapper is that it can't really resolve shell variables (and IIRC it can't resolve ~ either)

Reply:
Now that I think about what I just said: the wayland-ozone trick might not actually work with the binary wrapper, as it's using shell-only stuff.... I'll have to revert the chane it in this case.
I think I'll wait until someone actually has issues with darwin to use the binary wrapper.